### PR TITLE
chore: resolve vulnerabilities in deps

### PR DESCRIPTION
**Pull request by svlapin**
chore: resolve vulnerabilities in deps and add `security-check` as travis step

Checklist:

- [ ] Test added / Snapshots updated
- [ ] Documentation added / updated

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js: '8'
 cache: yarn
+before_script:
+  - npm install -g yarn@1.12.3
```
Comment (yamafaktory) [inline .travis.yml]: I think that we don't need to install Yarn on Travis CI, it's available by default (https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn).

Comment (svlapin) [reply .travis.yml]: It is, but in order to use `yarn audit` we need `>=1.12.3`, and on travis it's `1.3.2`

**Comment by yamafaktory** [comment]
Alright, then that make sense 👍. However, what about using yarn anyway for
the commands instead?

**Comment by svlapin** [comment]
@yamafaktory 
you mean doing `yarn security-check` instead of `npm run security-check` ? I thought about it, but decided to be consistent with `npm run test` and `npm run lint` already there.
Would you prefer to replace them with `yarn`, too?

**Comment by yamafaktory** [comment]
@svlapin yes exactly! Since we are using it by default on our projects, I guess this would make sense.

**Pull request merged**